src/ReactAddToCalendar.js

```diff
@@ -42,7 +42,9 @@ export default class ReactAddToCalendar extends React.Component {
     let showOptions = !this.state.optionsOpen;
 
     if (showOptions) {
-      document.addEventListener("click", this.toggleCalendarDropdown, false);
+      document.addEventListener("click", this.toggleCalendarDropdown, {
+        capture: true
```

Comment:
I can confirm this line fixes the issue. All other changes seem to be cosmetic code styling changes.

Reply:
It needs to be added on removeEventListener as well. 
The `UNSAFE_componentWillMount` will remove deprecation warning to prepare for react 18